src/index.html

```diff
@@ -1,14 +1,60 @@
-<!doctype html>
+<!DOCTYPE html>
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta
+      name="viewport"
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" >
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
+
   </head>
-  <body>
-    <h1>Moyo header</h1>
+  <body id = "home">
+    <header class="header">
+      <a href="#home" class="header__logo-link">
+        <img
+          src="./images/logo.png"
+          alt="MOYO logo"
+          class="header__logo"
+          >
+      </a>
+      <nav class="header__nav">
+        <ul class="header__nav-list">
+          <li class="header__nav-item">
+            <a href="#apple" class="header__nav-link is-active">apple</a>
+          </li>
+          <li class="header__nav-item">
+            <a href="#samsung" class="header__nav-link">samsung</a>
+          </li>
+          <li class="header__nav-item">
+            <a href="#smartphones" class="header__nav-link">smartphones</a>
+          </li>
+          <li class="header__nav-item">
+            <a
+             href="#laptops&computers" class="header__nav-link"
```

Comment:
move `class="header__nav-link"` on the new line

Reply:
done